src/App.tsx

```diff
@@ -17,18 +72,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                selectedOption={selectedOption}
+                setSelectedOption={setSelectedOption}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isTodosLoaded ? (
+                <Loader />
+              ) : (
```

Comment:
This is a little more readable

```suggestion
              {!isTodosLoaded
                ? (
                  <Loader />
                )
                : (
```